### ISSUE TITLE
.rodata highlighting

### ISSUE THREAD

**Issue by tstivers1990** [13xforever/language-x86_64-assembly#6]
I notice that .rodata doesn't get the same highlighting as sections like .data, .bss, and .text get.

For example:

```
section .bss ; highlighted

section .rodata ; not highlighted
```

I also notice if you end a section label with :, which is perfectly valid syntax, it doesn't get highlighted either.

For example:

```
section .bss ; highlighted

section .bss: ; not highlighted
```

**Comment by 13xforever** [comment]
Should be fixed now in 2.2.1

**Comment by 13xforever** [comment]
Let's try this again. Fixed in 55b4549
